website/docs/r/subscription.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Base"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_subscription"
+description: |-
+  Manages a Subscription.
+---
+
+# azurerm_subscription
+
+Manages a Subscription.
+
+~> **NOTE:** Destroying a Subscription controlled by this resource will place the Subscription into a cancelled state. It is possible to re-activate a subscription within 90-days of cancellation, after which time the Subscription is irrevocably deleted and the Subscription ID cannot be re-used. For further information see [here](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/cancel-azure-subscription#what-happens-after-subscription-cancellation)
```

Comment:
also worth mentioning it's possible to actually "Delete" a subscription 3 days after it's cancelled, but TF can't do this for users